deepchecks/checks/performance/simple_model_comparison.py

```diff
@@ -127,90 +180,89 @@ def _find_score(self, train_ds: Dataset, test_ds: Dataset, task_type: ModelType,
-            ratio = result['ratio']
-            if ratio < min_allowed_ratio:
-                return ConditionResult(False,
-                                       f'The given model performs {_more_than_prefix_adder(ratio, self.maximum_ratio)} '
-                                       'times compared to the simple model using the given scorer')
+        def condition(result: Dict, max_ratio=self.maximum_ratio, class_list=classes) -> ConditionResult:
+            scores_df = result['scores']
+            task_type = result['type']
+            metrics = scores_df['Metric'].unique()
+
+            metrics_dfs = []
+            if task_type == ModelType.MULTICLASS:
+                if class_list is None:
+                    class_list = scores_df['Class'].unique()
+                for metric, clas in itertools.product(metrics, class_list):
+                    score_rows = scores_df[(scores_df['Metric'] == metric) & (scores_df['Class'] == clas)]
+                    metrics_dfs.append(score_rows)
+            else:
+                for metric in metrics:
+                    score_rows = scores_df[(scores_df['Metric'] == metric)]
+                    metrics_dfs.append(score_rows)
+
+            not_passing_metrics = set()
+            for df in metrics_dfs:
+                origin_score = df[df['Type'] == 'Origin']['Value'].iloc[0]
+                simple_score = df[df['Type'] == 'Simple']['Value'].iloc[0]
+                ratio = get_scores_ratio(simple_score, origin_score, max_ratio)
+                if ratio < min_allowed_ratio:
+                    not_passing_metrics.update(df['Metric'].tolist())
+
+            if not_passing_metrics:
+                not_passing_metrics = sorted(not_passing_metrics)
+                msg = f'Metrics with scores ratio lower than threshold: {", ".join(not_passing_metrics)}'
+                return ConditionResult(False, msg)
             else:
                 return ConditionResult(True)
 
         return self.add_condition(f'Ratio not less than {format_number(min_allowed_ratio)} '
-                                  'between the given model\'s result and the simple model\'s result',
+                                  'between the given model\'s score and the simple model\'s score',
                                   condition)
 
 
-class _DummyModel:
-    @staticmethod
-    def predict(a):
-        return a
+class RandomModel:
+    """Class used to randomly predict from given series of labels."""
+
+    def __init__(self, labels):
+        self.labels = labels
+
+    def predict(self, a):
+        return np.random.choice(self.labels, a.shape[0])
+
+    def predict_proba(self, a):
+        classes_num = self.labels.unique().shape[0]
+        predictions = self.predict(a)
+
+        def create_proba(x):
+            proba = np.zeros(classes_num)
+            proba[x] = 1
+            return proba
+        return np.apply_along_axis(create_proba, axis=1, arr=predictions)
+
+
+class ScaledModel:
```

Comment:
way to do this is with [pipelines](https://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html)
Have a look at other usage in the package